### DISCUSSION TITLE
Can't add Radarr or Sonarr service

### DISCUSSION THREAD

**Discussion by richard-lee-863** [sct/overseerr#2191] [Help] [answered]
I am using dediseedbox service and have got Radarr and Sonarr currently connected to Ombi v3 no problems, all installed on the seedbox. 
Tried to setup services for Overseerr and get a "failed to connect" error. 

`2021-10-13T00:26:44.121Z [error][Radarr]: [Radarr] Failed to retrieve profiles: Request failed with status code 400 `

I'm using the same IP address, port and base url as i am in Ombi, which connects fine. 
Has anyone got this working for seedboxes?

**Comment by TheCatLady** [comment]
Are you running v3 Radarr/Sonarr?  Only v3 is supported by Overseerr, so one thing to check is that your *arrs are up-to-date.

**Reply by richard-lee-863** [reply]
Yep both running v3. Radarr = 3.2.2.5080, Sonarr = 3.0.6.1276

**Comment by richard-lee-863** [comment]
I notice that to connect to my Raddarr instance i need to access it via https://seedboxHostHere:PORT. 
If i just try plain HTTP i get the following error in the browser:
`The plain HTTP request was sent to HTTPS port`
However Overseerr seems to be hardcoding **http://** in front of the hostname/ip address field. 
I don't have SSL configured so i can't check that option. Why cant the http:// part be free text entry....?

**Reply by TheCatLady** [reply]
Check the `Use SSL` checkbox... 😉

**Reply by richard-lee-863** [reply]
Haha i already have many times, still no luck.
Does the Radarr / Sonarr service need to have SSL enabled as well for that to work though?

**Reply by richard-lee-863** [reply]
The Radarr / Sonarr apps do not have SSL enabled, but they still need to be accessed via the seedbox URL https://hostname:port. The port in the URL is different to the port in the Radarr / Sonarr settings though. But i suspect that is handled by the seedbox port rules

**Reply by TheCatLady** [reply]
The `Use SSL` checkbox changes the `http://` to `https://`, which is what you're looking to do.  The port likely needs to be `443` for HTTPS.

**Comment by richard-lee-863** [comment, marked as answer]
After correspondence from the seedbox provider, it appears that you cant use the URL Base path settings in Overseerr settings. 
I can now connect using the following settings:


Hostname or IP Address: xxxx.dediseedbox.com
Port: (port provided by seedbox)
Use SSL: **Yes**
API Key: My API Key From Sonarr
URL Base: **Leave this field empty**
